caffe2/core/dispatch/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+set(LIB_SOURCES
+        dummy.cpp
+)
+
+set(TEST_SOURCES
+        dummy_test.cpp
+)
+
+add_library(c10_dispatch OBJECT ${LIB_SOURCES})
```

Comment:
As @Yangqing said, can we make this just dispatch?

Reply:
Changed it. I'll keep `c10_utils` as `c10_utils` for now though, I'm afraid there might be another "utils" defined in one of the c2 or aten or pytorch or whatever CMakeLists.txt files and it would clash. We can rename it once everything is merged.